app/src/forms/form/controller.js

```diff
@@ -104,6 +105,7 @@ module.exports = {
   createSubmission:  async (req, res, next) => {
     try {
       const response = await service.createSubmission(req.params.formVersionId, req.body, req.currentUser);
+      emailService.submissionReceived(req.params.formId, response.id, req.headers.referer).catch(() => {});
```

Comment:
Typo - referrer, not referer. Definitely need to make sure we're not forcing API calls to use the wrong spelling.